content/general/concepts/unit-testing/unit-testing.md

```diff
@@ -0,0 +1,34 @@
+---
+Title: 'Unit Testing'
+Description: 'Unit testing involves testing ] parts of code to verify its correctness.'
+Subjects:
+  - 'Computer Science'
+  - 'Code Foundations'
+Tags:
+  - 'Debugging'
+  - 'Development'
+CatalogContent:
+  - 'learn-javascript-unit-testing'
+---
+
+Unit testing is an essential part of the coding process to verify all parts of code work correctly.
+
+A part or unit of code can be anything from a class function to a small snippet of code that performs a specific task. By testing individual components, it makes it easier to find exactly where an error lies.
+
+There are usually two types of unit testing — manual and automated testing.
+
+## Manual Testing
+
+Manual testing is when tests are being done manually by people. It can involve clicking on objects, providing input, or scrolling through items.
+
+Depending on the length of the software, this type of testing can be a lengthy process. A faster alternative to this would be automated testing.
```

Comment:
I was thinking about different ways to say "lengthy process."

> Depending on the length of the software, this type of testing may be time-consuming. A quick alternative to unit testing is automated testing.

I was thinking of removing the comparison. If you decide to leave the comparison in, I'd just keep it how you currently have it instead of the word I replaced it with ❗Maybe we could just say:

> An alternative to unit testing would be automated testing.

Reply:
I'll use "time-consuming" instead and use your second sentence instead.

Reply:
```suggestion
Depending on the length of the software, this type of testing can be time-consuming. An alternative to unit testing would be automated testing.
```